Realm.podspec

```diff
@@ -1,9 +1,10 @@
 Pod::Spec.new do |s|
   s.name                    = 'Realm'
+  s.module_map              = 'Realm/module.modulemap'
```

Comment:
I'd prefer to have this with the other build settings stuff rather than the metadata stuff.